src/lib.rs

```diff
@@ -77,7 +78,14 @@ fn add_mermaid(content: &str) -> Result<String> {
 
         // We're in the code block. The text is what we want.
```

Comment:
```suggestion
        // We're in the code block. The text is what we want.
        // Code blocks can come in multiple text events.
```

Reply:
Let's call this out in the comment, so I don't have to remember that when I read this in 3 months